ocs_ci/deployment/mce.py

```diff
@@ -356,3 +368,352 @@ def wait_mce_resources(self):
                 **depl_ocp_obj.get(retry=60, wait=10, dont_raise=True)
             )
             deployment_obj.wait_for_available_replicas(timeout=600)
+
+    def upgrade_mce(self):
```

Comment:
What is the difference between this method and ACMUpgrade class (https://github.com/red-hat-storage/ocs-ci/blob/33b598393dc677bc7a68b0a458742551222f1822/ocs_ci/ocs/acm_upgrade.py#L26)?